tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -33,6 +33,7 @@ describe('Acceptance: brocfile-smoke-test', function() {
   });
 
   afterEach(function() {
+    runCommand.killAll();
```

Comment:
interim solution to ensure we kill any leaked `runCommand` processes, once this PR lands. I will invest time in improving this area in https://github.com/ember-cli/ember-cli/issues/9073 after this PR lands.